### PR TITLE
Added: OSS-friends page

### PR DESCRIPTION
**Pull request by rohitmondal03**
Fix: #7 
Added "OSS friends page"

### CODE REVIEW

File: package-lock.json
Comment (Siddhant-K-code) [inline package-lock.json:1]: Can you please revert this file change 🙏🏼

Comment (rohitmondal03) [reply package-lock.json:1]: Ok I'll do that....

**Comment by rohitmondal03** [comment]
@Siddhant-K-code is it working good or is there any other change needed ?

**Comment by Nishchit14** [comment]
@rohitmondal03 Here is the preview, https://firecamp-io-git-fork-rohitmondal03-add-oss-frie-e4c9be-firecamp.vercel.app/oss-friends

The header is not visible.

image

**Review by Nishchit14 — CHANGES_REQUESTED** [review]
PR and Page design look great 👏 

I only suggested some naming convention changes, please follow them. rest is good to merge.

**Comment by rohitmondal03** [comment]
> @rohitmondal03 Here is the preview, https://firecamp-io-git-fork-rohitmondal03-add-oss-frie-e4c9be-firecamp.vercel.app/oss-friends
> 
> The header is not visible.
> 
> image
> 

What header should I give here ??
I have followed how the formbricks website have done !!

**Comment by Nishchit14** [comment]
image


@rohitmondal03 
you can change the mode from the footer dropdown.


Here is the component
https://github.com/firecamp-dev/website/blob/main/components/mode-toggle.tsx

**Review by Nishchit14 — APPROVED** [review]
LGTM :)

File: app/oss-friends/page.tsx [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+
+type OSSFriend = {
+    href: string;
+    name: string;
+    description: string;
+};
+
+// Mock Array from API
+const ossFriendsArray: OSSFriend[] = [
```
[the remark this answers is not in the document]
Comment (Nishchit14) [reply app/oss-friends/page.tsx]: @rohitmondal03 is it pending?

**Pull request merged**